src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+
+        Machine bulldozer = new Bulldozer();
+
+        Machine truck = new Truck();
+
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
```

Comment:
we can also remove the separate machine variables and initialize the right inside the array initialization, but I`d say that it is up to you and whatever approach you prefer